javascript-source/core/commandCoolDown.js

```diff
@@ -103,6 +103,7 @@
             if (command.equals('adventure')) {
                 if (cooldown[command] !== undefined) {
                     if (cooldown[command].time - $.systemTime() >= 0) {
+                        $.say($.whisperPrefix(username) + $.lang.get('adventuresystem.adventure.cooldown', parseInt(parseInt(cooldown[command].time - $.systemTime())/1000)));
```

Comment:
We do not want to force a cooldown message, plus `!yourbotname togglecooldownmessage` will enable the cooldown message for all command. We do not really want to have a bunch or cooldown messages for X command with a toggle.